tests/test_config.py

```diff
@@ -1,21 +1,33 @@
+from io import StringIO
 from unittest import TestCase
+from unittest.mock import patch
 
 from mlx.warnings import DoxyChecker, JUnitChecker, SphinxChecker, WarningsPlugin, XMLRunnerChecker
 
 
 class TestConfig(TestCase):
     def test_configfile_parsing(self):
-        warnings = WarningsPlugin(config_file="tests/config_example.json")
-        warnings.check('testfile.c:6: warning: group test: ignoring title "Some test functions" that does not match old title "Some freaky test functions"')
-        self.assertEqual(warnings.return_count(), 0)
-        warnings.check('<testcase classname="dummy_class" name="dummy_name"><failure message="some random message from test case" /></testcase>')
-        self.assertEqual(warnings.return_count(), 0)
-        warnings.check("/home/bljah/test/index.rst:5: WARNING: toctree contains reference to nonexisting document u'installation'")
-        self.assertEqual(warnings.return_count(), 1)
-        warnings.check('This should not be treated as warning2')
-        self.assertEqual(warnings.return_count(), 1)
-        warnings.check('ERROR [0.000s]: test_some_error_test (something.anything.somewhere)')
-        self.assertEqual(warnings.return_count(), 1)
+        warnings = WarningsPlugin(verbose=True, config_file="tests/config_example.json")
+        with patch('sys.stdout', new=StringIO()) as verbose_output:
+            warnings.check('testfile.c:6: warning: group test: ignoring title "Some test functions" that does not match old title "Some freaky test functions"')
+            self.assertEqual(warnings.return_count(), 0)
+            warnings.check('<testcase classname="dummy_class" name="dummy_name"><failure message="some random message from test case" /></testcase>')
+            self.assertEqual(warnings.return_count(), 0)
+            deprecation_warning = 'sphinx/application.py:402: RemovedInSphinx20Warning: app.info() is now deprecated. Use sphinx.util.logging instead.'
+            warnings.check(deprecation_warning)
+            self.assertEqual(warnings.return_count(), 0)  # ignored because of configured "exclude" regex
```

Comment:
Can we duplicate this test where we add additional `config_example_exclude.json`, just so that we can confirm functionality without exclude regex for all these checkers.

Reply:
Okay, here I added back the original unit test: cf767ff94eda158eb512d33d33069c5dc104acce. The exclude key is completely optional. It can be left out or have an empty list `[]` as value.